lib/core/GridBlock.js

```diff
@@ -47,20 +47,20 @@ class GridBlock extends React.Component {
     );
   }
 
-  renderBlockImage(image, imageLink) {
+  renderBlockImage(image, imageLink, imageAlt) {
     if (image) {
       if (imageLink) {
         return (
           <div className="blockImage">
             <a href={imageLink}>
-              <img src={image} />
+              <img src={image} alt={imageAlt} />
```

Comment:
maybe we do a check here in case `imageAlt` isn't provided?

like

```
<img src={image} alt={imageAlt ? imageAlt : ''}
```

or something like that?

Reply:
I think it's fine as-is because React does not render any props that are `undefined` or `null`.

Reply:
👍 Yeah I didn't add the check, because without an `imageAlt`, the `alt` didn't show up on inspection in the browser. Thanks @yangshun for explaining why!